client/phase.py

```diff
@@ -53,3 +39,18 @@ def subscription_handler(event_count, update_nr, subscription_id):
         print('NEXT PHASE!')
         client.next_phase()
         COUNT = 0
+
+
+def subscription_handler(event_count, update_nr, subscription_id):
+    global COUNT
```

Comment:
You don't need to redefine the global here.

Reply:
i think I do. if I dont do this, I get a "Unresolved reference 'COUNT' " in line 46

Reply:
This is the weirdest concept I have ever heard, but it seems you are right.